app/views/planning_applications/new.html.erb

```diff
@@ -7,6 +7,6 @@
     <h1 class="govuk-heading-l govuk-!-margin-top-7">
       New Application: Residential Lawful Development Certificate
     </h1>
-    <%= render 'form', planning_application: @planning_application %>
+    <%= render 'new_form', planning_application: @planning_application %>
```

Comment:
why not update `_form.html.erb`?

Reply:
have re-instated the old form and added a partial depending on whether the request is for "new"/"create" or "edit"